vss-processor-plugin/src/main/java/org/eclipse/kuksa/vssprocessor/plugin/VssProcessorPlugin.kt

```diff
@@ -67,15 +67,15 @@ class VssProcessorPlugin : Plugin<Project> {
             val buildDirPath = buildDir.absolutePath
             val vssDir = "${rootDir}${fileSeparator}$VSS_FOLDER_NAME"
 
-            val provideVssDefinitionTask =
-                project.tasks.register<ProvideVssDefinitionTask>(PROVIDE_VSS_DEFINITION_TASK_NAME) {
+            val provideVssFilesTask =
+                project.tasks.register<ProvideVssFilesTask>(PROVIDE_VSS_FILES_TASK_NAME) {
                     val searchPath = extension.searchPath.get().ifEmpty { vssDir }
-                    val vssDefinitionFilePath = StringBuilder(buildDirPath)
+                    val vssFilePath = StringBuilder(buildDirPath)
                         .append(fileSeparator)
                         .append(KSP_INPUT_BUILD_DIRECTORY)
                         .append(fileSeparator)
                         .toString()
-                    val vssDefinitionBuildFile = File(vssDefinitionFilePath)
+                    val vssBuildFile = File(vssFilePath)
```

Comment:
Somehow this name is confusing. Any specific reason for the "Build" in the name? I think it would be more fitting without it. Probably the "Build" wanted to say something more like "vssFileInBuildFolder" (don't use that :D) or similar.

Reply:
Probably outdated naming from some old state :) Changed it